css/properties/font-size-adjust.json

```diff
@@ -18,7 +18,7 @@
               "version_added": null
             },
             "edge": {
-              "version_added": false
+              "version_added": "12"
```

Comment:
Not sure about this one. Some hints suggest that `false` is correct:
Caniuse says false
Chrome status says "no public signals"
https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/6514821-font-size-adjust-other-font-properties is still open.

Reply:
This is a bit weird, noticed https://github.com/foolip/mdn-bcd-collector/issues/12 when digging into this.

I have to look into this more, but might not have time today.

Reply:
By https://wpt.fyi/results/css/css-fonts?q=font-size-adjust&run_id=5156214107275264&run_id=5100393054863360&run_id=5124297332883456&run_id=5651371558699008 it looks like this isn't *actually* supported by Edge. I loaded a few of the tests to make sure they're failing for good reasons and not because of something silly.

This is a case where there were contradictory signals from the tests I had, so I'll look into https://github.com/foolip/mdn-bcd-collector/issues/12 and make sure no tests are automatically updated in a case like this again.

Reply:
@Elchi3 I'll revert this change, but do you want to have a note saying that feature detection is broken? Something machine readable would be ideal, but at the very least with a note a future PR like this would be easier to judge.

Reply:
"notes" are exposed to MDN wiki tables and generally target web devs and not BCD contributors, but I think we could formulate a note here that is useful web devs who may use feature detection? 
```
"version_added": false,
"notes": "Although <code>'fontSizeAdjust' in document.body.style</code> returns <code>true</code>, this feature isn't supported."
```

Reply:
I've pushed a commit with that note, but `element.style` instead of `document.body.style` to not assume which element is used for testing. (`document.body` is null until the `<body>` element is created by the HTML parser, importantly including when running scripts in `<head>`.)